src/applications/rated-disabilities/actions/index.js

```diff
@@ -71,13 +71,28 @@ export function fetchTotalDisabilityRating(recordAnalyticsEvent = recordEvent) {
     dispatch({
       type: FETCH_TOTAL_RATING_STARTED,
     });
-    const response = await getData('/disability_compensation_form/rating_info');
-    const source = response?.sourceSystem;
-    const sourceString = source ? ` - ${source}` : '';
-    const apiName = `GET disability rating${sourceString}`;
 
-    const error = getResponseError(response);
-    if (error) {
+    try {
```

Comment:
The side effect of re-throwing the error is that we have to deal with it here now by using a try-catch